web/app/js/components/ResourceDetail.jsx

```diff
@@ -154,16 +154,10 @@ export class ResourceDetailBase extends React.Component {
 
     let apiRequests =
       [
-      // inbound stats for this resource
+        // inbound stats for this resource
```

Comment:
Mind removing the added whitespace?

Reply:
Sure, no problem, done. I thought introducing a whitespace was ok when done for consistency reasons. Please note below how that comment is the only one differently indented.

https://github.com/linkerd/linkerd2/blob/c078b4ff8d6f1e40a4eabcb520b2116fa4a52454/web/app/js/components/ResourceDetail.jsx#L156-L175